### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-04-18

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [space-ros/space-ros#150] [General]
Previous agenda: #144

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
- [x] Review status of [next release](https://github.com/orgs/space-ros/projects/3/views/8)
- [x] Open next meeting discussion

**Comment by xfiderek** [comment]
I'd love to discuss https://github.com/space-ros/space-ros/discussions/148 if time permits. It should not take longer than 5 mins

**Comment by Bckempa** [comment]
Next meeting agenda posted at #152
